pkg/loader/load.go

```diff
@@ -86,6 +86,9 @@ type loaderImpl struct {
 	// value can be tidb or mysql
 	saveAppliedTS           bool
 	lastUpdateAppliedTSTime time.Time
+
+	ctx    context.Context
```

Comment:
Add a TODO that we shouldn't store the `ctx` in the struct and should be refactored in the future etc.